less/bottom-content.less

```diff
@@ -0,0 +1,267 @@
+.home .bottom-content {  
+    padding-bottom: 10%;
+    padding-top: 10%;
+
+    @media @phone {
+      margin: 0 -8%;
+    }
+  
+    h1 {
+      .sub-header();
+      
+      
+
+      @media @phone {
+        padding-left: 7%;
+      }
+    }
+    
+    .location-box {
+      display: flex;
+      flex-direction: column;
+      justify-content: flex-start; 
+      padding-top: 10%; 
+          
+  
+  
+      .overlay-1 {
+        display: flex;
+        justify-content: flex-end;
+        align-items: center;
+        height: 34vh;      
+        background-image: url('../img/home/home-villas-img.png');
+        background-repeat: no-repeat;    
+        background-position: center;
+        background-size: cover;
+        margin-bottom: 5%;  
+        
+        @media @phone {
+          justify-content: center;
+          margin-bottom: 20%;
+        }
+
+        @media @desktop1100 {
+          margin-bottom: 8%;
+        }
+
+        @media @tablet {
+          margin-bottom: 15%;
+        }
+  
+        h1 {
+          background-color: @project-gray;                    
+          border: 1px solid @project-gray;
+          padding: 40px 60px; 
+          margin-top: 20%;                
+          font-weight: normal;
+          letter-spacing: 10px;  
+          
+          @media @phone {
+            margin-top: 60%;
+          }
+
+          @media @desktop1200 {
+            margin-top: 22%;
+          }
+
+          @media @desktop1100 {
+            margin-top: 25%;
+          }
+
+          @media @tablet {
+            margin-top: 36%
+          }
+        } // h1
+      } // .overlay
+    } // .location-box-1
+  } // .bottom-cont
+  
+  .location-box p { 
+    margin: 0 15%;      
+  }
+
+  
+  .bottom-content .location-box .overlay-2 {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    height: 34vh;      
+    background-image: url('../img/home/home-outskirts-img.png');
+    background-repeat: no-repeat;    
+    background-position: center;
+    background-size: cover;
+    margin-bottom: 5%;
+
+    @media @phone {
+      justify-content: center;
+      margin-bottom: 20%;
+    }
+
+    @media @desktop1100 {
+      margin-bottom: 8%;
+    }
+
+    @media @tablet {
+      margin-bottom: 15%;
+    }
+  
+    h1 {
+      background-color: @project-gray;      
+      border: 1px solid @project-gray;
+      padding: 40px 60px; 
+      margin-top: 20%; 
+      font-weight: normal; 
+      letter-spacing: 10px; 
+      
+      @media @phone {
+        margin-top: 60%;
+      }
+      @media @desktop1200 {
+        margin-top: 22%;
+      }
+
+      @media @desktop1100 {
+        margin-top: 25%;
+      }
+
+      @media @tablet {
+        margin-top: 36%
+      }
+    } // h1
+  } // .overlay-2
+  
+  .bottom-content .location-box .overlay-3 {
```

Comment:
In theory all of the "overlay" divs should share most of the same properties. So just use a single class for them. For the middle one, you can have a separate class where you change the position to be on the opposite side.